File: src/main/java/com/somefrills/features/tweaks/CameraTweaks.java
package com.somefrills.features.tweaks;

import com.somefrills.config.FrillsConfig;
import com.somefrills.config.tweaks.TweaksCategory.CameraTweaksConfig;
import com.somefrills.features.core.PassiveFeature;

public class CameraTweaks extends PassiveFeature {
    private final CameraTweaksConfig config;

    public CameraTweaks() {
        super();
        config = FrillsConfig.instance.tweaks.cameraTweaks;
    }

    public boolean clip() {
        return config.clip;
    }
}

File: src/main/kotlin/com/somefrills/features/tweaks/CameraTweaks.kt
package com.somefrills.features.tweaks

import com.somefrills.config.FrillsConfig
import com.somefrills.config.tweaks.TweaksCategory.CameraTweaksConfig
import com.somefrills.features.core.FrillsFeature
import com.somefrills.features.core.PassiveFeature

@FrillsFeature
class CameraTweaks : PassiveFeature() {
    private val config get() = FrillsConfig.tweaks.cameraTweaks

    fun clip(): Boolean {
        return config.clip
    }
}
